website/docs/r/spring_cloud.html.markdown

```diff
@@ -0,0 +1,63 @@
+subcategory: "App Platform"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud"
+sidebar_current: "docs-azurerm-resource-spring-cloud"
+description: |-
+  Manage Azure Spring Cloud Service instance.
+---
+
+# azurerm_spring_cloud
+
+Manage Azure Spring Cloud Service instance. Azure Spring Cloud provides a managed service that enables Java developers to easily build and run Spring-boot based microservices on Azure with no code changes.
+Within an azure spring cloud service, users can manage config server, manage multiple spring cloud apps and manage deployments
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "resGroup"
+  location = "Southeast Asia"
+}
+
+resource "azurerm_spring_cloud" "example" {
+  name                     = "sc-example"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+
+  tags = {
+    environment = "staging"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Service resource. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal. Changing this forces a new resource to be created.
+
+* `location` - (Optional) The GEO location of the resource. Changing this forces a new resource to be created. Until November 20, 2019 4:20 PM, only `East US`, `Southeast Asia`, `West Europe`, `West US 2` following locations are supported.

```

Comment:
We are past this date, is this still true?

Reply:
yes, it still only support these four locations.
In the azure portal, it's the same